src/vs/editor/test/common/controller/cursor.test.ts

```diff
@@ -2146,64 +2156,6 @@ suite('Editor Controller - Regression tests', () => {
 		});
 	});
 
-	test('issue Microsoft/monaco-editor#108 part 1/2: Auto indentation on Enter with selection is half broken', () => {
```

Comment:
Move it to indentation related test suite and uses indent rules instead of mocking `enterAction`.